evm-contracts/test/v0.6/BlockhashStore.test.ts

```diff
@@ -0,0 +1,132 @@
+import {
```

Comment:
This would run much faster in golang. :)

Reply:
It certainly would. Takes 10s to run these on my machine... 😞